core/modules/user/user.module

```diff
@@ -136,6 +136,23 @@ function user_entity_info() {
   );
 }
 
+/**
+ * Implements hook_layout_context_info().
+ */
+function user_layout_context_info() {
+  $info['user'] = array(
+    'title' => t('User'),
```

Comment:
Shouldn't we use "User account" here ? ( Source: https://github.com/backdrop/backdrop-issues/issues/127 )

Reply:
Yes! nice catch :)

Reply:
fixed.